test_conformance/vulkan/test_vulkan_api_consistency_for_3dimages.cpp

```diff
@@ -162,47 +162,6 @@ int test_consistency_external_for_3dimage(cl_device_id deviceID,
     test_error(errNum, "Unable to create Image with Properties");
     image.reset();
 
-    // Passing properties, image_desc and image_format all as NULL
-    image = clCreateImageWithProperties(context, NULL, CL_MEM_READ_WRITE, NULL,
```

Comment:
Same comment as above.

Reply:
This test case is ambiguous.  both format and image desc are null, either case is an error, and the OpenCL spec does specify if CL_INVALID_IMAGE_FORMAT_DESCRIPTOR  or CL_INVALID_IMAGE_DESCRIPTOR is to be returned if both are  NULL.

This test case does not make sense for this extension, there is no information being passed that assosciates clCreateImageWithProperties and cl_khr_external_memory.  

If we believe this is important coverage, it would be a better fit in api or basic.